src/server/game/Spells/Spell.h

```diff
@@ -274,13 +274,21 @@ class TC_GAME_API SpellCastTargets
         std::string m_strTarget;
 };
 
+enum SpellValueCritFlag : uint8
```

Comment:
I think you got that wrong tkrokli. He means not to store flags eg:  1, 2, 4, 8, 16.